src/Microsoft.DotNet.Cli.Utils/Command.cs

```diff
@@ -21,151 +18,38 @@ internal class Command
         private readonly StreamForwarder _stdOut;
         private readonly StreamForwarder _stdErr;
 
-        private bool _running = false;
+        private bool _running;
 
-        private Command(string executable, string args)
+        private Command(ProcessStartInfo psi)
```

Comment:
I really like this idea of taking ProcessStartInfo as arg. It would be incredibly cool if we could merge some of these with System.Diagnostics.Process